config/jenkins-jobs_infra.ci.jenkins.io.yaml

```
@@ -151,6 +151,8 @@ jobsDefinition:
       infra-reports:
         jenkinsfilePath: Jenkinsfile_updatecli
         disableTagDiscovery: true
+      stats.jenkins.io:
+        disableTagDiscovery: true
```

This setup uses `Jenkinsfile` as default. But we don't want to rebuild the whole [`Jenkinsfile`](https://github.com/jenkins-infra/stats.jenkins.io/blob/main/Jenkinsfile).

As such, I propose you to use the same pattern as other updatecli jobs:
```suggestion
      stats.jenkins.io:
        jenkinsfilePath: Jenkinsfile_updatecli
        disableTagDiscovery: true
```

Then, once merged, no build will be run for this new job until you add a `Jenkinsfile_updatecli` in a PR of https://github.com/jenkins-infra/stats.jenkins.io